src/style.css

```diff
@@ -1 +1,104 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+.body {
+  font-family: Roboto, serif;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  justify-content: start;
+  align-items: flex-start;
+  height: 100vh;
+  width: 100%;
+}
+
+.search-panel {
+  position: relative;
+  width: 100%;
+  max-width: 900px;
+  padding: 20px;
+}
+
+.search-input {
+  width: 100%;
+  font-size: 16px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  transition: all 0.3s ease;
+  box-shadow: 0 4px 4px 0 #00000040;
+  gap: 20px;
```

Comment:
The `gap` property is used here, but it doesn't apply to non-flex or non-grid containers. Consider removing it or applying it to a parent element that uses flex or grid layout.